opencensus/trace/exporters/jaeger_exporter.py

```diff
@@ -0,0 +1,451 @@
+
+    :type host_name: str
+    :param host_name: (Optional) The host name of the Jaeger HTTP Thrift.
+
+    :type port: int
+    :param port: (Optional) The port of the Jaeger HTTP Thrift.
+
+    :type endpoint: str
+    :param port: (Optional) The endpoint of the Jaeger HTTP Thrift .
+
+    :type agent_host_name: str
+    :param agent_host_name: (Optional) The host name of the Jaeger-Agent.
+
+    :type agent_port: int
+    :param agent_port: (Optional) The port of the Jaeger-Agent.
+
+    :type username: str
+    :param username: (Optional) The user name of the Basic Auth
+                     if authentication is required.
+
+    :type password: str
+    :param password: (Optional) The password of the Basic Auth
+                     if authentication is required.
+
+    :type transport: :class:`type`
+    :param transport: Class for creating new transport objects. It should
+                      extend from the base :class:`.Transport` type and
+                      implement :meth:`.Transport.export`. Defaults to
+                      :class:`.SyncTransport`. The other option is
+                      :class:`.BackgroundThreadTransport`.
+    """
+
+    def __init__(
+            self,
+            service_name='my_service',
+            host_name=None,
+            port=None,
+            endpoint='',
+            agent_host_name=DEFAULT_HOST_NAME,
+            agent_port=DEFAULT_AGENT_PORT,
+            agent_endpoint=DEFAULT_ENDPOINT,
+            username='',
+            password='',
+            transport=sync.SyncTransport):
+        self.transport = transport(self)
+        self.service_name = service_name
+        self.host_name = host_name
+        self.agent_host_name = agent_host_name
+        self.port = port
+        self.agent_port = agent_port
+        self.endpoint = endpoint
+        self.username = username
+        self.password = password
+        self.agent_client = self._agent_client
+        self.collector = self._collector
+
+    @property
+    def _agent_client(self):
+        return AgentClientUDP(
+            host_name=self.agent_host_name,
+            port=self.agent_port)
+
+    @property
+    def _collector(self):
+        if self.host_name is None or self.port is None:
+            return None
+
+        thrift_url = 'http://{}:{}{}{}'.format(
+            self.host_name,
+            self.port,
+            self.endpoint,
+            DEFAULT_ENDPOINT)
+
+        auth = None
+        if self.username != '' and self.password != '':
+            auth = (self.username, self.password)
+
+        return Collector(thrift_url=thrift_url, auth=auth)
+
+    def emit(self, span_datas):
+        """
+        :type span_datas: list of :class:
+            `~opencensus.trace.span_data.SpanData`
+        :param list of opencensus.trace.span_data.SpanData span_datas:
+            SpanData tuples to emit
+        """
+        trace = span_data.format_legacy_trace_json(span_datas)
```

Comment:
Thanks! You are actually the first person to make a new exporter since I made that change. I should probably go back and update some of the other exporters for reference.